src-docs/src/components/guide_section/guide_section.tsx

```diff
@@ -176,49 +183,60 @@ export const GuideSection: FunctionComponent<GuideSection> = ({
   };
 
   return (
-    <div className="guideSection" id={id}>
-      <GuideSectionExampleText title={title} wrapText={wrapText}>
-        {text}
-      </GuideSectionExampleText>
-
-      {renderingPlayground && (
-        <EuiFlyout
-          onClose={() => setRenderingPlayground(false)}
-          size="l"
-          paddingSize="none"
-          closeButtonPosition="outside"
-        >
-          {renderPlayground()}
-        </EuiFlyout>
-      )}
-      {(demo || fullScreen) && (
-        <>
-          <EuiSpacer />
-          <GuideSectionExample
-            example={
-              <EuiErrorBoundary>
-                {/* eslint-disable-next-line no-nested-ternary */}
-                {fullScreen == null ? (
-                  <div>{demo}</div>
-                ) : demo == null ? (
-                  <EuiButton
-                    fill
-                    iconType="fullScreen"
-                    href={`#${path}/${fullScreen.slug}`}
-                  >
-                    Fullscreen demo
-                  </EuiButton>
-                ) : (
-                  demo
-                )}
-              </EuiErrorBoundary>
-            }
-            tabs={renderTabs()}
-            ghostBackground={ghostBackground}
-            demoPanelProps={demoPanelProps}
-          />
-        </>
-      )}
-    </div>
+    <EuiPanel
+      color={color || 'transparent'}
```

Comment:
[optional nit] if you already have a `color = 'transparent'` fallback above, I don't think you need this
```suggestion
      color={color}
```

Reply:
Ahh, actually I forgot to remove the default value because I add extra padding only if a color is actually passed down.